### DISCUSSION TITLE
break combo or prevent combo from increasing on a meh/50 judgement in osu!mania

### DISCUSSION THREAD

**Discussion by iSimpFor** [ppy/osu#13155] [Ideas]
a meh/50 in osu!mania is considered a poor judgement as you would have to be way off to get this judgement so it's suggested to have a combo break or to prevent combo from increasing on a meh/50 to encourage people to get better judgements

this would mean getting a meh/50 in osu!mania would prevent a full combo from happening but it would a osu!mania fc more valuable 

this is also considering that fact that osu!mania has 6 judgements rather than 2 or 3

**Comment by peppy** [comment]
Logically this makes very little sense. Strongly against this personally.

**Comment by smoogipoo** [comment]
May be an interesting way to counter mashing, though I’m not sure how often mehs contribute instead of goods.

Wouldn’t be against trying this/something similar, especially with scoring mods I have planned.

**Comment by peppy** [comment]
For what it's worth, I have no issue with this (combo break version) if it is consistent across all game modes. ie. "meh" in osu! would also be a miss. It could be renamed to "poor" or something more suiting, as was historically the case with betamania and other titles that had such a judgement.

Basically we would have two levels of "miss", one where it told you if you were too early or late.

**Comment by enoslayd** [comment]
I kinda agree with the "prevent increase of combo" (as well for other rulesets).

**Comment by peppy** [comment]
That's something I won't budge on sorry. Either miss or combo increase, choose one.

**Comment by enoslayd** [comment]
Then combo increase. Though this should be a community poll or something. Giving meh a miss treatment means entirely removing meh judgement

**Comment by iSimpFor** [comment]
meh would still give a higher accuracy rating than a miss, it would just break combo instead
